api/room/schema.py

```diff
@@ -20,6 +21,16 @@ class Meta:
         model = RoomModel
 
 
+def save_room_tags(room, room_tags):
+    for tag in room_tags:
+        room_tag = TagModel.query.filter_by(id=tag).first()
+        if not room_tag:
+            room.delete()
```

Comment:
Please share the reason for the .delete() because the method name implies that saving is being carried out.

Reply:
@RonKbS, this is because whether the tag id is valid or not, the room is still saved (kindly test this locally). I therefore have to remove it if the mutation has an invalid tag.

Reply:
@bevkololi please respond to @RonKbS

Reply:
I see you've pushed some changes and removed the room.save() on line 102 just before this function is called. The .delete() method should be unnecessary now.

Reply:
It still saves the room @RonKbS. That line did not affect it.

Reply:
Worked on this. Thanks.